### DISCUSSION TITLE
【TreeSelect】希望能新增一个TreeSelect组件

### DISCUSSION THREAD

**Discussion by kailong321200875** [element-plus/element-plus#1146] [Ideas]
希望能新增一个TreeSelect组件，这个组件盼望已久了，之前是自己结合Select 和 Tree 组件来使用，希望官方大大可以考虑新开发下这个组件，就使用场景来说，还是很多地方都会用到的。
参考：https://2x.antdv.com/components/tree-select-cn/

**Comment by Ryan2128** [comment]
WIP, #878

**Reply by cjq02** [reply]
都一年了还是没看到

**Comment by 7086cmd** [comment]
级联不行吗

**Reply by cjq02** [reply]
级联不能全部展示末节点下的子孙节点，只能一级一级展示
